app/scripts/controllers/push-platform-notifications/services/services.ts

```diff
@@ -10,9 +10,14 @@ const url = process.env.PUSH_NOTIFICATIONS_SERVICE_URL;
 const REGISTRATION_TOKENS_ENDPOINT = `${url}/v1/link`;
 const sw = self as unknown as ServiceWorkerGlobalScope;
 
+export type RegToken = {
+  token: string;
+  platform: 'extension' | 'mobile' | 'portfolio';
+};
```

Comment:
New request/response shape, just needed updating of the implementation and tests.

Reply:
It's probably not worth using a constant, right?

Reply:
yah, we can extract this to shared libraries in future :P